### PR TITLE
seed for random generator

### PR DESCRIPTION
**Pull request by fflori4n**
Added method for setting seed of random generator

## Connection with issue(s)

Resolve issue #25 

## Testing and Review Notes

Only tested in Flutter for the web, but should work for all.

## To Do

- [ ] double check the original issue to confirm it is fully satisfied
- [ ] add testing notes and screenshots in PR description to help guide reviewers

### CODE REVIEW

File: lib/src/internet.dart [hunk 1]
```diff
@@ -152,25 +149,4 @@ class Internet {
...
-      .element(UserAgents.fromJson(userAgentDatas)
-          .userAgents
-          .map((e) => e)
-          .where(
-            (element) =>
-                element.osName.toLowerCase().contains(osName.toLowerCase()),
-          )
-          .toList())
-      .userAgent
-      .toString();
```
Comment (drager) [inline lib/src/internet.dart:175]: Why are these being deleted?

File: lib/src/faker.dart [hunk 2]
```diff
@@ -39,7 +37,6 @@ class Faker {
         currency = const Currency(),
         food = const Food(),
         guid = const Guid(),
-        image = const Image(),
```
Comment (drager) [inline lib/src/faker.dart:42]: Why remove `image`?

**Review by drager — CHANGES_REQUESTED** [review]

File: lib/src/internet.dart (hunk 1 above)
Comment (fflori4n) [reply lib/src/internet.dart:175]: Yep. My bad. It seems that I used and older fork of faker, without the userAgent and image commit. Can you omit those changes, or should I make another pull request? Only added a method in random_generator.dart

Comment (drager) [reply lib/src/internet.dart:175]: Please make a new commit that brings those deletions back if possible.

**Comment by drager** [comment]
Closing this due to #32

**Pull request closed without merging**